mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -271,6 +275,8 @@ CLASS net/minecraft/class_1309 net/minecraft/entity/LivingEntity
 	METHOD method_36977 damageHelmet (Lnet/minecraft/class_1282;F)V
 		ARG 1 source
 		ARG 2 amount
+	METHOD method_37119 tickItemStack (Lnet/minecraft/class_1799;)V
```

Comment:
```suggestion
	METHOD method_37119 tickItemStackUsage (Lnet/minecraft/class_1799;)V
```
Since in server player entity, it triggers an advancement criteria trigger and this is called when the given stack is used.